society-of-biblical-literature-fullnote-bibliography.csl

```diff
@@ -1078,13 +1078,27 @@
               <text variable="genre" text-case="capitalize-first"/>
             </if>
           </choose>
-          <group delimiter=". ">
-            <text macro="originally-published"/>
-            <group delimiter=", ">
-              <text macro="reprint"/>
-              <text macro="publisher"/>
-            </group>
-          </group>
+          <!-- If no original-date is present, use a semicolon in the bibliography, rather than a period. Doing so allows original-publisher and original-publisher-place to be used to handle co-publication situations where both publishers are equally original. See SBLHS2 §4.1.4.4 and CMOS17 §14.140. -->
```

Comment:
Can you remove this. We don't want to include the workaround in the version of the style used in the repo.